### ISSUE TITLE
Reviews count from DevHub is not consistent with reviews count in new frontend

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-server#7540]
Follow up of [#7111](https://github.com/mozilla/addons-server/issues/7111)
Prerequisites: you need to have at least one add-on submitted; add-on contains empty user reviews

STR:

1. Log in to Developer Hub and go to Manage my submissions
2. Look at an add-on that has been rated and also includes some empty user reviews - make a note of the number of reviews displayed in dev-hub
3. Click on the "<#  reviews>" link to read all the available reviews on the frontend

Actual result:
The number of reviews displayed in DevHub is not the same as the actual number of reviews (empty+written) listed on the frontend

Expected result:
Number of reviews displayed in DevHub matches the actual number of reviews (empty+written) listed on the new frontend

Notes:
Reproduced on all AMO servers

[dev-hub]
![image](https://user-images.githubusercontent.com/31961530/79117996-4b96e200-7d95-11ea-9b0d-76487a650d78.png)

[frontend]
![image](https://user-images.githubusercontent.com/31961530/79118007-505b9600-7d95-11ea-8d6b-b6eb8f8b76b8.png)

**Comment by diox** [comment]
Is it just a matter of displaying the number of reviews with text instead of all reviews ?

**Comment by jvillalobos** [comment]
Yeah, it sounds like it's counting both reviews and ratings, and it should only count reviews (with text).

**Comment by caitmuenster** [comment]
If this is your first contribution, please refer to https://wiki.mozilla.org/Add-ons/Contribute/Code on how to get started. 

Mentor: @eviljeff 

Andrew, can you add a few more steps on how to get started with this? :)

**Comment by eviljeff** [comment]
looks like to fix this you'll need to update two templates to use `addon.text_ratings_count` rather than `addon.total_ratings`
https://github.com/mozilla/addons-server/blob/master/src/olympia/ratings/templates/ratings/impala/reviews_link.html
https://github.com/mozilla/addons-server/blob/master/src/olympia/ratings/templates/ratings/reviews_link.html

This will likely result in some test fails - if you will need to fix the tests too.

**Comment by Vishal19111999** [comment]
@eviljeff I want to solve this as my first bug

**Comment by eviljeff** [comment]
@Vishal19111999 it's best practise to comment first to let us (and other potential contributors) know you're working on it.

**Comment by Vishal19111999** [comment]
@eviljeff Why is this shown as closed?

**Comment by eviljeff** [comment]
@Vishal19111999 you fixed it!

**Comment by Vishal19111999** [comment]
@eviljeff No, I didn't, It was fixed by probably @yatinmaan

**Comment by eviljeff** [comment]
@Vishal19111999 sorry, you're right.  I read both emails at the same time and thought you'd submitted the fix.

**Comment by Vishal19111999** [comment]
Can you assign me or just suggest a good first bug ?? It would be a great help...thanks

**Comment by eviljeff** [comment]
@Vishal19111999 we don't have any other definite good first bugs right now.  We have some "contrib: welcome" issues that we think could be solved by contributors, but they require someone who is more familiar with django and is willing to (self) learn how the addons-server codebase works.  (We can help explain the problem if the issue isn't clear, but aren't going to help you through developing the solution)

**Comment by Vishal19111999** [comment]
@eviljeff I have a month of experience in Django...I think with a little googling I would be able to figure things out...could you recommend any welcome bug to me??

**Comment by Vishal19111999** [comment]
@eviljeff I am facing problem setting things up...I have followed the install docks, but yet http://localhost:3000/en-US/firefox/ is returning a blank page...port 80 is also returning the default NginX page...port 8000 is not returning anything...port 3000 is returning 'Connection was reset'
![Screenshot from 2019-09-24 10-25-47](https://user-images.githubusercontent.com/29145212/65482170-c2ca5e80-deb5-11e9-8a80-b331afc37c29.png)
![Screenshot from 2019-09-24 10-26-04](https://user-images.githubusercontent.com/29145212/65482174-c8c03f80-deb5-11e9-8668-1b58a8870645.png)

**Comment by eviljeff** [comment]
As I said "we don't have any other good first bugs right now".
This issue is closed.  It's not the place to discuss unrelated set up problems.  https://wiki.mozilla.org/Add-ons/Contribute/Code talks about what to do and where to get guidance.

**Comment by AlexandraMoga** [comment]
Funny thing is that frontend is displaying empty reviews for some time now, but I forgot I had this issue opened 🙈

This means that, with this patch, the issue is reintroduced - DevHub being the one showing a smaller number of reviews this time. 

So, we might need to revert this patch after all...

**Comment by AlexandraMoga** [comment]
Re-opening because this is still an issue

**Comment by arjunvijayanathakurup** [comment]
Hey @AlexandraMoga, Can I work on this issue. I seem to notice that this was already fixed a year ago but I guess it's still relevant.

**Comment by eviljeff** [comment]
@arjunvijayanathakurup it's been re-opened because the behaviour changed (back) on the frontend so this needs to be changed back too.  It's pretty much doing the opposite of https://github.com/mozilla/addons-server/pull/12411/files (some of the other code might have changed in the meantime)

**Comment by leni1** [comment]
>  It's pretty much doing the opposite of https://github.com/mozilla/addons-server/pull/12411/files (some of the other code might have changed in the meantime)

@eviljeff so basically undoing all the commits made so far and pushing them to a new pull request :thinking: ?

**Comment by eviljeff** [comment]
> so basically undoing all the commits made so far and pushing them to a new pull request 🤔 ?

There was only a single commit.  The code needs to be changed back.

**Comment by Tintinator** [comment]
I can take this if @leni1 isn't on it.

**Comment by AlexandraMoga** [comment]
Verified fixed on -dev. DevHub and the frontend are showing the same number of reviews now (i.e. total ratings)

![image](https://user-images.githubusercontent.com/31961530/86778436-0a942e00-c063-11ea-9f32-173d07fa0e89.png)
